### DISCUSSION TITLE
Why Aren't There Any Social Media Logos?

### DISCUSSION THREAD

**Discussion by Pandeyashish17** [tailwindlabs/heroicons#970] [Ideas]
I've been using Heroicons for a while now and I've noticed that there aren't any icons for social media platforms like Facebook, Twitter, or Instagram. I was wondering if anyone knows why that is, and if there are any plans to add them in the future.

**Comment by zefir-git** [comment]
Why would you need these brand icons in Heroicons specifically? Why should Herocions add thousands of brand icons, then carefully watch if any of them changes to update it, ensure brand usage guidelines are followed, etc. etc. Especially since there already is a project dedicated to brand icons: https://simpleicons.org/
